app/views/tasks_views.py

```diff
@@ -0,0 +1,47 @@
+from flask import Blueprint, jsonify, request
+from ..db_services.tasks_service import get_all_tasks, create_task, get_task_by_id, update_task, delete_task,get_user_tasks
+from ..route.tasks import tasks_blueprint
+from flask_login import current_user
+
+
+@tasks_blueprint.route('/api', methods=["GET", "POST"])
+def manage_tasks():
+    if request.method == "GET":
+        tasks_list = get_all_tasks()
+        return jsonify(tasks_list=tasks_list)
+    elif request.method == "POST":        
+        try:          
+            data = request.get_json()           
+            user_id = data.get('user_id')          
+            result = create_task(title=data['title'], description=data['description'], user_id = current_user.id)
+            return jsonify(result)
+        except Exception as e:
+            print(str(e))
+            return jsonify(error="give data in JSON format in 'in raw'")
+
+@tasks_blueprint.route('/api/<string:task_id>', methods=["GET", "PUT", "DELETE"])
+def manage_task(task_id):
```

Comment:
No auth, Only the admin and the user or manager associated to that task should able to see or update the task